source/MaterialXGenShader/GenOptions.h

```diff
@@ -191,6 +192,10 @@ class MX_GENSHADER_API GenOptions
     /// Calculate fallback bitangents from existing normals and tangents
     /// inside the bitangent node.
     bool hwImplicitBitangents;
+
+    /// OSL-specific override to enable/disable the optional argument
+    /// to OSL's texture() function.
+    bool oslFileTextureColorspace;
```

Comment:
In order to make this new generator option as clear as possible, what would you think about naming it `oslTextureColorspace`?  The corresponding C++ constant could then be named `T_OSL_TEXTURE_COLORSPACE` for consistency.